internal/view/live_view.go

```diff
@@ -98,9 +98,15 @@ func (v *LiveView) ResourceChanged(lines []string, matches fuzzy.Matches) {
 		v.maxRegions = len(matches)
 		ll := make([]string, len(lines))
 		copy(ll, lines)
+		offsetForLine := make(map[int]int)
 		for i, m := range matches {
 			loc, line := m.MatchedIndexes, ll[m.Index]
-			ll[m.Index] = line[:loc[0]] + `<<<"search_` + strconv.Itoa(i) + `">>>` + line[loc[0]:loc[1]] + `<<<"">>>` + line[loc[1]:]
+			offset := offsetForLine[m.Index]
+			loc[0] += offset
+			loc[1] += offset
+			regionStr := fmt.Sprintf(`<<<"search_%d">>>%s<<<"">>>`, i, line[loc[0]:loc[1]])
```

Comment:
Not keen on this in terms of allocs. Prefer the old way (tho not as readable...)